src/main/resources/META-INF/plugin.xml

```diff
@@ -5,12 +5,42 @@
     <vendor>mitchellolsthoorn</vendor>
 
     <depends>com.intellij.modules.platform</depends>
+    <!-- add java functionality. See https://blog.jetbrains.com/platform/2019/06/java-functionality-extracted-as-a-plugin/ for more information -->
+    <depends>com.intellij.modules.java</depends>
+
+    <description>
+        <![CDATA[
+            <p>A plugin that uses EvoSuite to generate tests. Developed by TU Delft students.
```

Comment:
As a place holder this should be fine. If I understood correctly then this is our description/instruction about our plug-in and how to use it. We might want to see if it's possible to link a file here with our descriptions instead of writing descriptions directly. I think TestKnight will be a good place to check it out, because they have a fairly nice documentation on JetBrains market place.

Reply:
Indeed, we haven't looked into it yet, but will in the future, It will make more sense, i think, to add it in the end of the project, since then we know exactly which functionality do we have, and how everything works.